docs/querying/api.md

```diff
@@ -132,7 +132,7 @@ URL query parameters:
 - `query=<string>`: Prometheus expression query string.
 - `start=<rfc3339 | unix_timestamp>`: Start timestamp.
 - `end=<rfc3339 | unix_timestamp>`: End timestamp.
-- `step=<duration>`: Query resolution step width.
+- `step=<duration>`: Query resolution step width in seconds.
```

Comment:
Not quite. This is a <duration> as defined over in the configuration docs, but it also accepts a floating point number of seconds. We should copy the definition of <duration> over, and add that this can be a float too.

Reply:
Do you mean it's the same as this:
`<duration>: a duration matching the regular expression [0-9]+(ms|[smhdwy])`
?
I assumed it was seconds cause all example I've seen generally don't specify the unit, and if I understand correctly it defaults to seconds then.

Reply:
ok - so this PR is not necessary, I'm closing it

Reply:
It's not the same as a <duration>, as it also allows a floating point value.